packages/zowe-explorer-api/src/fs/types/abstract.ts

```diff
@@ -68,7 +68,6 @@ export type ConflictData = {
 export interface IFileSystemEntry extends vscode.FileStat {
     name: string;
     metadata: EntryMetadata;
-    type: vscode.FileType;
```

Comment:
that's smart! 
This is not a breaking change since vscode.FileStat already has a type property 👍